conf/modules.config

```diff
@@ -465,6 +465,34 @@ process {
         ]
     }
 
+    withName: ANGSD_DOCOUNTS {
+        ext.args = [
+            "-iCounts 1",
+            "-r ${params.contamination_estimation_angsd_chrom_name}:${params.contamination_estimation_angsd_range_pos1}-${params.contamination_estimation_angsd_range_pos2}",
+            "-minMapQ ${params.contamination_estimation_angsd_mapq}",
+            "-minQ ${params.contamination_estimation_angsd_minq}"
+        ].join(' ').trim()
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: ANGSD_CONTAMINATION {
+        publishDir = [
+            path: { "${params.outdir}/contamination_estimation/angsd/"},
+            mode: params.publish_dir_mode,
+            pattern: '*.txt'
+        ]
+    }
+
+    withName: PRINT_CONTAMINATION_ANGSD {
+        publishDir = [
+            path: { "${params.outdir}/contamination_estimation/angsd/"},
+            mode: params.publish_dir_mode,
+            pattern: '*.txt'
```

Comment:
To match behaviour of version (2.*), this should also publish the output json file.